packages/core/cli/src/scripts/generateTemplates/index.ts

```diff
@@ -1,5 +1,5 @@
-import getIntegrations from '@vue-storefront/cli/src/utils/getIntegrations';
-import createProject from '@vue-storefront/cli/src/scripts/createProject';
+import getIntegrations from '../../utils/getIntegrations';
```

Comment:
@Fifciu are you sure this will work? I remember you did that on purpose

Reply:
Looks fine for me. Anyway, I encourage to manual test built version.

Reply:
I've tested this many times already (and once more just a few minutes ago) and everything seems to still work. I asked @Fifciu to test it as well, just to be sure.

Reply:
I've just tested it and it looks fine :)